packages/dashboard/src/components/queryEditor/queryEditor.tsx

```diff
@@ -1,44 +1,93 @@
-import { type IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
+import { IoTSiteWiseClient } from '@aws-sdk/client-iotsitewise';
 import SpaceBetween from '@cloudscape-design/components/space-between';
+import Tabs from '@cloudscape-design/components/tabs';
+import TextContent from '@cloudscape-design/components/text-content';
+import { TwinMakerKGQueryDataModule } from '@iot-app-kit/source-iottwinmaker';
+import Box from '@cloudscape-design/components/box';
 import React, { useState } from 'react';
 
 import { QueryEditorErrorBoundary } from './queryEditorErrorBoundary';
 import { ResourceExplorer, ResourceExplorerProps } from './resourceExplorer';
 import { StreamExplorer } from './streamExplorer';
+import { AdvancedSearch } from './advancedSearch/advancedSearch';
+import { Property } from './advancedSearch/model/property';
 
 export interface QueryEditorProps {
   client: IoTSiteWiseClient;
+  // Enables Advanced Search if provided
+  kGDatamodule?: TwinMakerKGQueryDataModule;
 }
 
 /**
  * User interface element enabling users to manage their data source queries.
  *
  * @remarks
  *
- * At this highest level, the QueryEditor's responsibility is to manage the state synchronizing the
- * ResourceExplorer and StreamExplorer components.
+ * At this highest level, the Que lorer components.
  */
-export function QueryEditor({ client }: QueryEditorProps) {
+export function QueryEditor({ client, kGDatamodule }: QueryEditorProps) {
   const [selectedAssets, setSelectedAssets] = useState<ResourceExplorerProps['selectedAssets']>([]);
   const [assetId, setAssetId] = useState<string | undefined>(undefined);
 
+  // TODO: Use selectedProperties to add properties to dashboard when "Add" is clicked
+  const [selectedProperties, setSelectedProperties] = useState<Property[]>([]);
+
+  // TODO: Remove once we add the "add" button
+  console.log(selectedProperties);
+
   // these assets will be described and their asset properties listed by the stream explorer
   const selectedAssetIds = selectedAssets
     .filter((asset): asset is typeof asset & { id: string } => asset.id != null)
     .map(({ id }) => id);
 
-  return (
-    <QueryEditorErrorBoundary>
-      <SpaceBetween size='l'>
-        <ResourceExplorer
-          client={client}
-          selectedAssets={selectedAssets}
-          onSelect={setSelectedAssets}
-          assetId={assetId}
-          onChangeAssetId={setAssetId}
-        />
-        <StreamExplorer client={client} assetIds={selectedAssetIds} />
-      </SpaceBetween>
-    </QueryEditorErrorBoundary>
-  );
+  const resourceExplorerTab = {
+    label: 'Browse assets',
+    id: 'first',
+    content: (
+      <Box padding='s'>
+        <SpaceBetween size='m' direction='vertical'>
+          <TextContent>
+            <h3>Browse Assets</h3>
+          </TextContent>
```

Comment:
Why are you not using the Cloudscape `<Header />` component?

Reply:
Because I did not know about the Cloudscape `<Header /> ` component but I will add that in a followup PR.